assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -109,6 +109,9 @@ vm.dirty_background_ratio=3
 #> latency-performance
 vm.swappiness=10
 
+#> rps configuration
```

Comment:
Please add a note that we also configure this via a sysctl.d file

Reply:
Done. Thanks